tests/specs/future/unstable_flags/main.js

```diff
@@ -1,4 +1,3 @@
-console.log(typeof await navigator.gpu.requestAdapter() === "object"); // Throws without `--unstable-gpu`
```

Comment:
does it still fail with jammy?

Reply:
yes because the runner is on ubuntu 24 (only the sysroot is on jammy)